onadata/apps/logger/models/attachment.py

```diff
@@ -86,3 +87,14 @@ def file_hash(self):
     def filename(self):
         if self.media_file:
             return os.path.basename(self.media_file.name)
+
+    def soft_delete(self, user=None):
+        """
+        Soft deletes an attachment by adding a deleted_at timestamp.
+        """
+
+        soft_deletion_time = timezone.now()
+        self.deleted_at = soft_deletion_time
```

Comment:
I don't see the value in a separate variable here, why not put `timezone.now()` right here instead of assigning it to the var first?